src/Extensions/Hosting.Services.Web/HostBuilderExtensions.cs

```diff
@@ -17,39 +18,49 @@ public static class HostBuilderExtensions
 		/// Add Kestrel service listener to stateless service
 		/// </summary>
 		/// <typeparam name="TStartup">The type containing the startup methods for the web listener</typeparam>
-		public static ServiceFabricHostBuilder<StatelessServiceContext> AddKestrelListener<TStartup>(
-			this ServiceFabricHostBuilder<StatelessServiceContext> builder,
+		public static ServiceFabricHostBuilder<StatelessService> AddKestrelListener<TStartup>(
+			this ServiceFabricHostBuilder<StatelessService> builder,
 			string name,
 			ServiceFabricIntegrationOptions options,
 			Action<IWebHostBuilder>? builderExtension = null)
 				where TStartup : class =>
-					builder.AddKestrelListener<TStartup, StatelessServiceContext>(name, options, builderExtension);
+					builder.AddKestrelListener<TStartup,StatelessService,StatelessServiceContext>(
```

Comment:
>, [](start = 40, length = 1)

Add spaces after `,`

Reply:
Applies throughout

---
In reply to: [395664015](https://github.com/microsoft/Omex/pull/172#discussion_r395664015) [](ancestors = 395664015)

Reply:
Added, thank you